src/app/components/SkipToContentWrapper.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component, RefObject } from 'react';
+import scrollTo from '../../helpers/scrollTo';
+import { MAIN_CONTENT_ID, Provider } from '../context/SkipToContent';
+import HiddenLink from './HiddenLink';
+
+export default class SkipToContentWrapper extends Component {
+  private mainContent: RefObject<any> | undefined;
+
+  public render() {
+    return <Provider value={{registerMainContent: this.registerMainContent}}>
+      <HiddenLink onClick={this.scrollToTarget} href={`#${MAIN_CONTENT_ID}`} tabIndex={1}>Skip to Content</HiddenLink>
+      {this.props.children}
+    </Provider>;
+  }
+
+  private scrollToTarget = (event: React.MouseEvent<HTMLAnchorElement>) => {
+    if (this.mainContent) {
+      event.preventDefault();
+      scrollTo(this.mainContent);
```

Comment:
this doesn't seem to match src/helpers/scrollTo.ts

Reply:
oh never mind the second and third params are optional. do we really need those to be there?

Reply:
I removed them. Mostly because I wanted to keep code coverage @ 💯